content/docs/studio/how-it-works.md

```diff
@@ -1,19 +1,26 @@
 # How Iterative Studio Works
 
-[Iterative Studio](https://studio.iterative.ai/) works with the data, metrics
-and hyperparameters that you add to your ML project Git repositories. It works
-very closely with your Git ecosystem. Using DVC and Git, you will push all your
-ML experiments to your GitHub, GitLab or Bitbucket repositories as Git commits.
+[Iterative Studio](https://studio.iterative.ai/) works with the data, metrics,
+hyperparameters and model metadata that you add to your ML project Git
+repositories. It works very closely with your Git ecosystem. Using DVC and Git,
+you will push all your ML experiments to your GitHub, GitLab or Bitbucket
+repositories as Git commits. Similarly, using GTO, and possibly MLEM, you will
+push all your ML model details to the repositories as Git commits and Git tags.
```

Comment:
I think we should completely rewrite this paragraph now. Further patching it is making it too complicated. Also, it overlaps a lot with the intro in the [home page](https://dvc-org-mr-docs-ezydbpywaed4ti.herokuapp.com/doc/studio) (which also overlaps a little with [Prepare Your Repositories](https://dvc-org-mr-docs-ezydbpywaed4ti.herokuapp.com/doc/studio/user-guide/prepare-your-repositories)).

Maybe the home page should have a more general description like "Studio condenses a suite of tools from Iterative.ai to help you manage all the aspects of your ML projects on top of Git ..." or something like that.

Then this paragraph can be closer to what's in the home page and list+link all the tools used under the hood.

Reply:
Revised the index page and the how-it-works page. PTAL.